ray_julia_jll/gen/README.md

```diff
@@ -0,0 +1,25 @@
+# Building and Publishing JLLs
+
+1. Run the `make.jl` script to build the tarball for the host platform and a given julia-version.
+Builds are required for `linux x86_64` and `macos-arm64` on julia `v1.8.0` and `v1.9.0`.
```

Comment:
I believe you can use the latest version of this minor releases to make the builds?
```suggestion
Builds are required for `linux x86_64` and `macos-arm64` on julia `v1.8` and `v1.9`.
```

Reply:
I wasn't sure... in JuliaBinaryWrappers I never saw anything other than `1.X.0` being used...

e.g. https://github.com/JuliaBinaryWrappers/Geant4_julia_jll.jl/tree/main/src/wrappers

Reply:
Okay, We should call this out explicitly then. It's easy to miss

Reply:
So I did a quick check on this and on julia 1.9 I was able to download and use the 1.9.2 asset (and run tests) so I think we're actually ok to only specify the minor version

Reply:
kept it as `1.8` and `1.9`